apps/chat/src/components/Common/ApplicationWizard/CustomAppView.tsx

```diff
@@ -0,0 +1,264 @@
+import React, { useCallback, useMemo } from 'react';
+import { Control, Controller, useForm } from 'react-hook-form';
+
+import { useTranslation } from 'next-i18next';
+
+import classnames from 'classnames';
```

Comment:
```suggestion
import classNames from 'classnames';
```

it's better to have 1 import convention. Lets use `classNames` name as far as we use it in other components